### ISSUE TITLE
Could provide train/val split for ImageNet-21k?

### ISSUE THREAD

**Issue by mitming** [open-mmlab/mmclassification#698]
@mzr1996 @LXXXXR @xiaojieli0903 

Thanks for your contribution, I want to know if there are `train.txt`, `val.txt` for ImageNet-21k? like the annotation files for ImageNet-1k. Could you please provide it?

**Comment by Ezra-Yu** [comment]
here are the annotation file and script in [GoogleDrive](https://drive.google.com/file/d/10JbI8KatIn0yRoPmb91ULd0V56yoD0u-/view?usp=sharing).

ImageNet22k only has train.txt.

**Comment by mitming** [comment]
> here are the annotation file and script in [GoogleDrive](https://drive.google.com/file/d/10JbI8KatIn0yRoPmb91ULd0V56yoD0u-/view?usp=sharing).
> 
> ImageNet22k only has train.txt.

Thanks for your reply. Do you mean that ImageNet-22K only has a training set? Or do we need to look elsewhere for the validation set

**Comment by mitming** [comment]
> > > here are the annotation file and script in [GoogleDrive](https://drive.google.com/file/d/10JbI8KatIn0yRoPmb91ULd0V56yoD0u-/view?usp=sharing).
> > > ImageNet22k only has train.txt.
> > 
> > 
> > Thanks for your reply. Do you mean that ImageNet-22K only has a training set? Or do we need to look elsewhere for the validation set
> 
> There is no validation set in the ordinary ImageNet22k dataset, you can split the train set into two parts for your task.

Can I understand that the original ImageNet-22k only has a training set and a test set? Thanks a lot~

**Comment by Ezra-Yu** [comment]
Yes! Imagnet22k usually is used for pre-train and there is no need for a validation set when pre-training.

I will close this issue later, do you have other questions?

**Comment by mitming** [comment]
> Yes! Imagnet22k usually is used for pre-train and there is no need for a validation set when pre-training.
> 
> I will close this issue later, do you have other questions?

I have no further questions. Thanks again for your reply~
